src/stationd/stationd.py

```diff
@@ -134,22 +126,26 @@ def command_handler(
                     'rotator',
                     'sdr_b200',
                 ]:
-                    command_parser(getattr(self, device), command_obj)
-                    self.shared['num_active_ptt'] = command_obj.num_active_ptt
-                elif len(command_obj.command) == 1 and command_obj.command[0] == 'gettemp':
-                    read_temp(command_obj, self.pi_cpu)
+                    message = command_parser(getattr(self, device), command)
+                elif len(command) == 1 and command[0] == 'gettemp':
+                    message = read_temp(self.pi_cpu)
                 else:
-                    invalid_command_response(command_obj)
+                    message = 'FAIL: Invalid Command\n'
             except PTTConflictError:
-                ptt_conflict_response(command_obj)
-            except PTTCooldownError as e:
-                ptt_cooldown_response(command_obj, e.seconds)
-            except MollyGuardError:
-                molly_guard_response(command_obj)
+                message = f'FAIL: {" ".join(command)} PTT Conflict\n'
+            except amp.PTTCooldownError as e:
+                message = f'WARNING: Please wait {e.seconds} seconds and try again\n'
+            except amp.MollyGuardError:
+                message = 'Re-enter the command within the next 20 seconds to proceed\n'
```

Comment:
It'd be better (i think?) if the number of seconds to wait was dynamic so the user can more accurately determine when next to re-enter the command.  Seems like we should be able to easily add a `seconds` attribute to the MollyGuardError class (like what we have in `PTTCooldownError`)